blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScm.java

```diff
@@ -304,7 +304,7 @@ protected static HttpURLConnection connect(String apiUrl, String accessToken) th
             throw new ServiceException.PreconditionRequired("Github accessToken does not have required scopes. Expected scopes 'user:email, repo'");
         }
         if(status == 404){
-            throw new ServiceException.NotFoundException("Not Found");
+            throw new ServiceException.NotFoundException("Github could not be found. Remote server sent 404.");
```

Comment:
phrasing doesn't make sense - "github" can't be sound? is it meant to be github server?